<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudjettiDlg</class>
 <widget class="QDialog" name="BudjettiDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budjetti</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Tilikausi</string>
       </property>
       <property name="buddy">
        <cstring>tilikausiCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="tilikausiCombo"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Kohdennus</string>
       </property>
       <property name="buddy">
        <cstring>kohdennusCombo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="kohdennusCombo"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="view"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="ohjeNappi">
       <property name="text">
        <string>Ohje</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/ohje.png</normaloff>:/pic/ohje.png</iconset>
       </property>
       <property name="shortcut">
        <string>F1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="kopioiNappi">
       <property name="text">
        <string>Kopioi &amp;edellisestä</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/kopioi.png</normaloff>:/pic/kopioi.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="summaLabel">
       <property name="text">
        <string>   0,00  €</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="tallennaNappi">
       <property name="text">
        <string>&amp;Tallenna</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/ok.png</normaloff>:/pic/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="peruNappi">
       <property name="text">
        <string>&amp;Sulje</string>
       </property>
       <property name="icon">
        <iconset resource="../pic/pic.qrc">
         <normaloff>:/pic/peru.png</normaloff>:/pic/peru.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../pic/pic.qrc"/>
 </resources>
 <connections/>
</ui>
